examples/aws_container_app/README.MD

```diff
@@ -0,0 +1,84 @@
+# Terraform Module: AWS Integration Setup
+
+## Overview
+
+This Terraform module sets up the necessary AWS infrastructure for integrating with Port using ECS and Fargate. The module provisions an ECS cluster, ECS task definition, and other related resources.
+
+## Prerequisites
+
+- Ensure you have Terraform installed on your machine. You can download it from the [official Terraform website](https://www.terraform.io/downloads.html).
+- AWS CLI configured with appropriate credentials.
+- Ensure your AWS user has the necessary permissions to create and manage ECS, IAM, and VPC resources.
+
+## Installation
+
+1. **Clone the repository** containing the Terraform module.
+2. **Navigate to the directory** where the module is located.
+
+## Usage
+
+Create a `main.tf` file in your Terraform project and include the following code, modifying the values as necessary:
+
+```hcl
+module "aws_integration" {
+  source = "./path-to-your-module"
+
+  subnets             = ["subnet-1", "subnet-2", "subnet-3"]
+  port = {
+    client_id     = "<client-id>"
+    client_secret = "<client-secret>"
+  }
+  integration = {
+    identifier = "my-aws-integration"
+    config = {
+      live_events_api_key = "<your-api-key>"
+      organization_role_arn  = "arn:.../organization-role"
+      account_read_role_name = "ocean-integration-role"
+    }
+  }
+  cluster_name              = "port-ocean-aws-exporter"
+  vpc_id                    = "vpc-1"
+  initialize_port_resources = true
+  create_default_sg         = false
+  allow_incoming_requests   = true
```

Comment:
lets organize it in order of sections, port stuff first, real time events, then cloud configuration